index.html

```diff
@@ -9,13 +9,77 @@
 </head>
 
 <body>
-	<header>
-		<nav>
-			<p>Fix</p>
-		</nav>
+	<header class="site-header">
+
+		<div class="header-top">
+			<div class="container">
+				<div class="header-left">
+					<a class="connection" href="mailto:mhhasanul@gmail.com">
+						<img src="./images/icons/mail.svg" alt="mail">
+						mhhasanul@gmail.com
+					</a>
+
+
+					<a href="tel:(12345)67890" class="connection">
+						<img src="./images/icons/phone.svg" alt="(12345)67890">
+						(12345)67890
+					</a>
+				</div>
+				<div class="header-right">
+
+					<a href="#">English<img src="./images/icons/akar-icons_chevron-down.svg"
+							alt="akar-icons_chevron-down"></a>
```

Comment:
In the ALT we should describe our image instead of just copy name (this applies to all alt on the page)

Reply:
corrected